resources/translations/fi.edn

```diff
@@ -137,7 +137,7 @@
                           :submitted "%1 lähetti hakemuksen käsiteltäväksi."
                           :unknown "Tuntematon"}
                  :id "Tunniste"
-                 :intro [:div.intro [:p "Täyttämällä ja lähettämällä tämän hakemuksen kaikki hakijat voivat saada oikeuden haettuihin resursseihin. Kunkin hakijan on lisäksi hyväksyttävä käyttöehdot viimeistään hakemuksen käsittelyn jälkeen."]]
+                 :intro [:div.intro [:p "Täytä hakemus ja lähetä se käsiteltäväksi. Hakemukseen voi kutsua jäseniksi useampia hakijoita, ja jokaisen on hyväksyttävä käyttöehdot viimeistään kun hakemus on hyväksytty."]]
```

Comment:
I think the second clause in the last sentence is dependent on the first clause, so it would be better to use subordinate conjunction there, e.g., "jolloin", instead of "ja". (I think the English translation has a similar slight issue.)

Reply:
"…useampia hakijoita, joista jokaisen…" and "…viimeistään silloin, kun…"

Reply:
"joista jokaisen" is fine but "silloin" is redundant, and the comma was discussed above.

Reply:
Added "joista jokaisen", left out "silloin".